#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("min_x", double_t, 0, "", -2.0, -10.0, 10.0)
gen.add("max_x", double_t, 0, "", 2.0, -10.0, 10.0)
gen.add("min_y", double_t, 0, "", -2.0, -10.0, 10.0)
gen.add("max_y", double_t, 0, "", 2.0, -10.0, 10.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "HeightmapToPointCloud"))
